src/utils/getExpiredDaysAgo.js

```diff
@@ -0,0 +1,15 @@
+import { filter } from "lodash";
```

Comment:
Why using lodash's filter? Is it mandatory? Can't the thing be done with normal JS filter function? Please look into it once!